packages/grid/src/HeaderWrapper.tsx

```diff
@@ -0,0 +1,148 @@
+import React, {
+  useState,
+  useRef,
+  useEffect,
+  useCallback,
+  MutableRefObject,
+  MouseEvent,
+} from 'react';
+import { HeaderCellWrapper } from './HeaderCell';
+import { Header, MovingElem } from './styled';
+import { IHeaderWrapper, IColumn } from './interfaces';
+
+
+export const HeaderWrapper = ({
+  fullWidth,
+  translateX,
+  columns,
+  onChangeWidth,
+  onChangeMoving,
+  changeChangingColumns,
+  theme,
+  shouldMovingColumns,
+  shouldChangeColumnsWidth,
+}: IHeaderWrapper) => {
+  const mappedColumns = useRef(columns);
+  const [isMoving, changeIsMoving] = useState(false);
+  const clickX = useRef(0);
+  const movingColumnIndex = useRef<number>();
+  const movingColumnData = useRef<IColumn>();
+  const movingElemRect = useRef<ClientRect>();
+  const headerRef = useRef<HTMLDivElement>() as MutableRefObject<HTMLDivElement>;
```

Comment:
Why not just useRef<MutableRefObject<HTMLDivElement>>(); ?

Reply:
`useRef<HTMLDivElement>();` it will be more correct